src/mlpack/core/util/param.hpp

```diff
@@ -1223,51 +1258,10 @@ using DatasetInfo = DatasetMapper<IncrementPolicy, std::string>;
  * @param REQ Whether or not parameter is required (boolean value).
  */
 #ifdef __COUNTER__
-  #define PARAM_IN(T, ID, DESC, ALIAS, DEF, REQ) \
+  #define PARAM_COMPLETE(T, ID, DESC, ALIAS, NAME, REQ, IN, TRANS, DEF) \
```

Comment:
Up to you, but honestly I think it would be okay to just give this the simpler name of `PARAM`. :+1:

Reply:
I was just afraid that PARAM would be already taken up by some other macro but since the tests are passing now I think it is not and definitely better to use here.